src/minecraft/CommandHandler.js

```diff
@@ -5,7 +5,7 @@ class CommandHandler {
   constructor(minecraft) {
     this.minecraft = minecraft
 
-    this.prefix = '!'
+    this.prefix = minecraft.app.config.discord.prefix || "?"
```

Comment:
Rather than reusing the discord prefix it would be better to have a separate option in the configuration for the minecraft prefix to allow more customisation.